modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -110,6 +111,21 @@ export interface SelectBaseProps extends CoreSelectBaseProps {
    * @default true
    */
   isMenuAnimated: boolean;
+  /**
+   * If true, automatically flip the SelectBase menu to keep it visible if necessary (e.g., if the the menu would otherwise display below the visible area of the viewport).
+   * @default true
+   */
+  isMenuAutoFlipped: boolean;
```

Comment:
These config options read like state vs props. `isMenuAutoFlipped` vs `shouldMenuAutoFlip`.

Reply:
Renamed `isMenuAutoFlipped` to `shouldMenuAutoFlip`.